### PR TITLE
Support non-hbase-prefix configs

### PR DESCRIPTION
**Pull request by cowboy129**
1. support hbase configs such as zookeeper.znode.parent
2. avoid scala matcher error in HBaseAsyncWriteJournal

### CODE REVIEW

File: src/main/scala/akka/persistence/hbase/journal/HBaseAsyncWriteJournal.scala [hunk 1]
```diff
@@ -58,6 +58,7 @@ class HBaseAsyncWriteJournal extends Actor with ActorLogging
     flushWrites()
     Future.sequence(futures) map {
       case _ if publishTestingEvents => context.system.eventStream.publish(FinishedWrites(persistentBatch.size))
+      case _ =>
```
Comment (ktoso) [inline src/main/scala/akka/persistence/hbase/journal/HBaseAsyncWriteJournal.scala:61]: ouch, thanks!

File: src/test/resources/application.conf [hunk 2]
```diff
@@ -23,8 +23,8 @@ hbase-journal {
   publish-testing-events = on
 
   hbase {
-    cluster.distributed = false
-    zookeeper.quorum = "localhost:2181"
+    hbase.cluster.distributed = false
```
Comment (ktoso) [inline src/test/resources/application.conf:26]: uhm, the key of this value is now: `hbase.hbase.cluster.distributed` (it's in the hbase config block). 
Why this change?

Comment (cowboy129) [inline src/test/resources/application.conf:26]: I just want to put all hbase config under 'hbase' block. such as 
hbase.cluster.distributed
hbase.zookeeper.quorum
zookeeper.znode.parent

Because not all configs are prefix with 'hbase.', I write full name here.

Comment (ktoso) [inline src/test/resources/application.conf:26]: Uhm, but configs use scoping, so:

```
hbase {
  thing.another = 1
}
```

is equal to 

```
hbase.thing.another = 1
```

Notice the hbase scope on line 25.

Did something not work for you with this?

Comment (cowboy129) [inline src/test/resources/application.conf:26]:
``` scala
def getHBaseConfig(config: Config): Configuration
```

in HBaseJournalInit.scala will add 'hbase.' prefix for all configs under 'hbase' scope. so 

``` scala
hbase {
    zookeeper.znode.parent
}
```

will become `hbase.zookeeper.znode.parent` which is not correct

Comment (cowboy129) [inline src/test/resources/application.conf:26]: hi @ktoso . How's the change? need some more explanations or modifications?

Comment (ktoso) [inline src/test/resources/application.conf:26]: hey, seems good, but I just need some time to merge (not while running around the office :-)).
Thanks!

**Comment by ktoso** [comment]
Forgot about this PR, will pull it in and release `0.4.1`

**Comment by ktoso** [comment]
Merged manually, thanks a lot for the contrib! I must have been quite tired when reviewing previously.
Releasing as `0.4.1`
